validator/evaluation/eval_diffusion.py

```diff
@@ -0,0 +1,149 @@
+import base64
+import json
+import os
+from typing import Dict
+from typing import List
+from typing import Tuple
+
+from fiber.logging_utils import get_logger
+from validtor.core.models import Img2ImgPayload
+
+from utils import base64_to_image
+from utils import calculate_l2_loss
+from utils import download_from_huggingface
+from validator.core import constants as cst
+from validator.utils import comfy_api_gate as api_gate
+
+
+logger = get_logger(__name__)
+
+
+def load_comfy_workflows():
+    with open(cst.LORA_WORKFLOW_PATH, "r") as file:
+        lora_template = json.load(file)
+
+    return lora_template
+
+
+def edit_workflow(payload: Dict, edit_elements: Img2ImgPayload, text_guided: bool) -> Dict:
+    payload["Checkpoint_loader"]["inputs"]["ckpt_name"] = edit_elements.ckpt_name
```

Comment:
We really can put these into models to make it cleaner

I know you said it's ok on 19 but defo can make it clean

Reply:
Since we know what we're accessing